pandas/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+
+@pytest.fixture
+def datapath(request):
```

Comment:
> Do we know if any tests actually use the get_frame() stuff that get_data_path is doing?

I suppose this is just to figure out the "current file" of the test? So we can use relative paths instead of absolute (eg you changed `tm.get_data_path('multiindex_v1.pickle')``to `datapath('indexes', 'data', 'multiindex_v1.pickle')`). 
Not sure how important this is (eg when moving restructuring test dir layout, both have (dis)advantages)

Reply:
Sorry, I should have been clearer. I was wondering if there was a base class somewhere, and two children classes inheriting from that base class, so that `get_data_path()` might resolve to different directories. But I didn'mt see anything like that.

But indeed, when we move test files to different directories, we'll need to update the paths. I think that's fine since it'll be a simple find / replace.

Reply:
this should go in the conftest 1 level up